packages/evm/src/fungible.ts

```diff
@@ -0,0 +1,111 @@
+import type {
+  Domain,
+  EvmResource,
+  FeeHandlerType,
+} from "@buildwithsygma/sygma-sdk-core";
+import type { SecurityModel } from "../../core/src/index.js";
+
+/* eslint-disable @typescript-eslint/no-unused-vars */
+interface TransactionRequest {
+  to: string;
+  value: bigint;
+  data: string;
+  gasLimit: bigint;
+}
+
+interface Eip1193Provider {
+  request(request: {
+    method: string;
+    params?: Array<unknown> | Record<string, unknown>;
+  }): Promise<unknown>;
+}
+
+type EvmFee = {
+  fee: bigint;
+  type: FeeHandlerType;
+  handlerAddress: string;
+  tokenAddress?: string;
+  percentage?: number;
+  minFee?: bigint;
+  maxFee?: bigint;
+};
+
+/**
+ * Return amount of liquidity tokens on resource handler
+ * @param provider
+ * @param resource
+ */
+export function getLiquidity(
+  provider: Eip1193Provider,
```

Comment:
shouldn't this method be a part of `EvmFungibleAssetTransfer` class? Previously it was part of `BaseAssetTransfer` class. 

It requires `Config` in its context to be able to extract handler address and then query balance of that address

Reply:
Does it make sense to pass `Config` as a param to this function?

Reply:
@MakMuftic can you point me to its use case/usage atm? then i'd be able to come up with a refined interace